### PR TITLE
#160933361 User can sign up on the app

### PR DESCRIPTION
**Pull request by daniellamarr**
#### What does this PR do?
- This PR creates a user account on the app
#### Description of Task to be completed?
- When a user navigates to **signup.html**, the user should be able to signup on the app
- All fields should be required
- A successful signup should return a status of 201
#### How should this be manually tested?
- Pull this branch **ft-user-able-signup-front-end-160933361** off this repository
- Navigate to **signup.html** on the branch
- Fill in all required fields and submit
- A successful signup redirects user to login page.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[160933361](https://www.pivotaltracker.com/story/show/160933361)
#### Questions:
None

### CODE REVIEW

File: js/auth.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            password: document.getElementById('password').value,
+            cpassword: document.getElementById('cpassword').value
+        }
+        return form;
+    }
+    signupForm.addEventListener("submit", (e)=>{
+        e.preventDefault();
+        trigger(signup());
+    });
+}
```
Comment (KaiserPhemi) [inline js/auth.js]: #### Nitpick
Always insert a line after the last line in a file.

File: js/class.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                'Accept': 'application/json'
+            },
+            body,(res) => {
+                if (res==201) {
+                    setTimeout(()=>{window.location.href='login.html'},3000)
+                }
+            }
+        );
+    }
+}
```
Comment (KaiserPhemi) [inline js/class.js]: #### Nitpick
Insert a new line here.

File: js/class.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
+const path = 'https://daniellamarr-fast-food-fast.herokuapp.com';
+
+class FastFood {
```
Comment (KaiserPhemi) [inline js/class.js:10]: #### Critical
Document your classes and methods using `jsdoc`
Check [here](http://usejsdoc.org/about-getting-started.html)

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

File: js/class.js (hunk 3 above)
Comment (daniellamarr) [reply js/class.js:10]: Done

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**